### DISCUSSION TITLE
Live streaming [ video playing --> media recorder --> recorded blob --> deepgram live -->  net0001 issue ]

### DISCUSSION THREAD

**Discussion by vinitk34** [deepgram/community#454] [General help]
Hello everyone ,
I am trying to transcribing a video for my application [ vuejs + nodejs]

Initially,  i am playing a video and recording data using mediaRecorder

let recorder = new MediaRecorder (this.videoStream);
recorder.ondataavailable = (event) => this.dataBlob.push(data) ;

then am creating blob and sending it to server...

let recordedBlob - new Blob(this.dataBlob, ( type: 'video/webm' ]):
this.socket.emit(`sendDataToServe', ( data: recordedBlob ));

Now on server side

const deepgramLive = deepgram.transcription
					 .live(
						lang: "en-US"
						model: "nova",
						encoding :"vp8"',
						sample_rate :48000,
						endpointing :false,
					  }

deepgramLive.addListener("open", (evt) => (
console.log( 'OPEN ?????????');
}
deepgramLive.addListener("close", (err) -> E
console.log("close ??????' , err);
}:
deepgramLive.addListener("transcriptReceived" > (transcription) -> t
console. log( "&&&&&&&&&&&&&::::: REcieved ??????????`, transcription):
}

In some fuctions i am calling

deepgramLive.send(data);

.. with same data in prerecorded functionality. i am getting the result but with live transcription 
i am getting  net0001 error

![error](https://github.com/deepgram/community/assets/66453657/d6459eea-293b-4611-ba3b-25ec789bb210)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by vinitk34** [comment]
Some more info

1) While using preRecorded() transcription => I am sending recorded blob from mediaRecorder() on every 5 sec  and sending it to deepgram
									and it was giving result . but the reason i moved to live transcription . that if any word
									is between 0-5 sec and 5-10 sec .then the word is split in two parts and the intent of 
                                                                        word is lost
											 
									 
2) While using deepgramLive transcription => I am sending recorded blob on every 250ms and sending it to deepgram but sometime it give 
                                                                          result but it give result for around 3sec of data after that it shows error net0001 shown above .
                                                                          DeepgramLive Instance and listener is created once . and DeepgramLive.send(data) is hit on 
                                                                          recieving datablob from frontend.
                                                                          I checked the recorded blob type .It was vp8/opus . So i tried changing different encoding as well

**Reply by lukeocodes** [reply]
If you’re receiving a 1011 code with a NET-0000 payload, Deepgram wasn’t able to generate a transcript for the audio data you’re sending before the system timed out.

This occurs when you open the socket, before you start sending audio through. If you get the error, then it is because Deepgram is not receiving any data.

You can choose to only open the socket when the microphone opens, or keep the socket open with a KeepAlive request.

https://dpgr.am/net0001

**Reply by vinitk34** [reply]
Hi ,
I am receiving a 1011 code with NET-0001.
I am not sending data from the microphone . I am sending recorded blob to live transcription API and keeping socket open with a keepalive request.

**Reply by davidvonthenen** [reply]
@vinitk34 if you are sending data either the blobs are malformed or you aren't selecting the correct audio encoding when establishing the connection.

**Reply by vinitk34** [reply]
Thank you @dvonthenen for the reply ,
So as per your reply, we can use Deepgram's live transcription API with continuous input of 250ms "video" blobs each with correct encoding ?

Information:- I tried with different encoding and was able to get transcription of around 3.3s of video only once each time and then 2nd transcription recieved :
{Type : Metadata , transaction_key: deprecated} 
After that connection gets close with
Reason:''
Code:1000

So , is it possible that the problem is still with the encoding ??
